docs/notes/bugfix-15865

```diff
@@ -0,0 +1 @@
+# Fixed Dictionary example for is within
```

Comment:
@bhall2001 I think you've used the same bugfix note description as for the other PR!

Also, could you make sure the file has a `.md` suffix please?